locales/en-US/app.ftl

```diff
@@ -227,10 +227,15 @@ Home--menu-button-instructions =
     Enable the profiler menu button to start recording a performance
     profile in { -firefox-brand-name }, then analyze it and share it with profiler.firefox.com.
 
-Home--addon-button = Install add-on
-Home--addon-button-instructions =
-    Install the Gecko Profiler Add-on to start recording a performance
-    profile in { -firefox-brand-name }, then analyze it and share it with profiler.firefox.com.
+Home--enable-button-unavailable =
+    .title = This profiler instance was unable to connect to the WebChannel, so it cannot enable the profiler menu button.
```

Comment:
I assume `WebChannel` should not be translated? If that's correct, we should add a comment asking to not translate it.

Reply:
I've added a comment.